recipes-tag/edrixs/meta.yaml

```
@@ -0,0 +1,48 @@
+{% set name = "edrixs" %}
+{% set version = "0.0.4" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 4d2f0757406dd01197925586db6a9363e2f3945f208fae5dc98e947139d8df9e
+
+build:
+  number: 0
+
+requirements:
+  host:
+    - python
+    - setuptools
+    - numpy
+  build:
+    - arpack-mpi
+    - libopenblas
+    - openmpi
+    - {{ compiler('fortran') }}
+  run:
+    - arpack-mpi
+    - libopenblas
+    - matplotlib
+    - mpi4py
```

@shenmidelin, I think we can use it right here, so that `conda install edrixs` will work without `nomkl` package:
```suggestion
    - mpi4py
    - nomkl
```

I will try it first, and will incorporate after the successful testing.

---

Great! Thanks! @mrakitin

---

I actually tried to install the generated package with `conda install -c local edrixs` with and without `nomkl`, and the both [run_rixs_pysolver.py](https://github.com/NSLS-II/edrixs/blob/master/examples/more/RIXS/LaNiO3_thin/run_rixs_pysolver.py) and [run_rixs_fsolver.py](https://github.com/NSLS-II/edrixs/blob/master/examples/more/RIXS/LaNiO3_thin/run_rixs_fsolver.py) worked for both cases. Was it another example which failed?

I think linking is done during the building part. Should `nomkl` be in the `host` or `build` requirements instead of the proposed solution in https://github.com/mrakitin/lightsource2-recipes/pull/1?

---

Let me try it again, I will give you feedback later.

---

Thanks! We can actually build it "as is", and then reiterate via another PR if you wish to save time on building `arpack-mpi`.

---

Yes, you are right, there is no problem for `examples/more/RIXS/LaNiO3_thin`. The error is from another example `examples/more/ED/14orb`.  I got error like this:

`[7daf078951ff:06992] *** Process received signal ***`
`[7daf078951ff:06992] Signal: Segmentation fault (11)`
`[7daf078951ff:06992] [ 1] /home/builder/.conda/envs/edrixs/lib/python3.7/site-packages/numpy/../../../libmkl_intel_lp64.so(zdotc+0xba)[0x7faaa9885d9a]`
`[7daf078951ff:06992] [ 2] /home/builder/.conda/envs/edrixs/lib/python3.7/site-packages/edrixs/../../../libparpack.so.2(pzneupd_+0x175d)[0x7faa9932eb6d]`

To reproduce this error, 

`cd examples/more/ED/14orb`
`./get_input.py`
`./run_fedsolver.py`

---

I tested it with the `nomkl` package added to the list of `run` dependencies, and it worked fine!